README.md

```diff
@@ -11,7 +11,7 @@
 [![Anaconda-Server Badge](https://anaconda.org/conda-forge/esmvaltool/badges/version.svg)](https://anaconda.org/conda-forge/esmvaltool)
 ![stand with Ukraine](https://badgen.net/badge/stand%20with/UKRAINE/?color=0057B8&labelColor=FFD700)
 
-![esmvaltoollogo](https://github.com/ESMValGroup/ESMValTool/blob/main/doc/sphinx/source/figures/ESMValTool-logo-2.png)
+![esmvaltoollogo](https://github.com/ESMValGroup/ESMValTool/blob/cec27e25c2e07975290769a9d1822befd88792b2/doc/sphinx/source/figures/ESMValTool-logo-2.png)
```

Comment:
```suggestion
![esmvaltoollogo](https://raw.githubusercontent.com/ESMValGroup/ESMValTool/main/doc/sphinx/source/figures/ESMValTool-logo-2.png)
```
I'm not too sure about pointing to a specific commit.

Reply:
See also https://stackoverflow.com/questions/41983209/how-do-i-add-images-to-a-pypi-readme-that-works-on-github

Reply:
Cheers, bud, was looking for the raw permalink myself, that's what Github gave me though, it's being silly